src/spanningtrees/kruskal.jl

```diff
@@ -0,0 +1,55 @@
+immutable KruskalHeapEntry{T}
+    edge::Pair{Int, Int}
+    dist::T
+end
+
+isless(e1::KruskalHeapEntry, e2::KruskalHeapEntry) = e1.dist < e2.dist
+
+""" Performs Quick-Find algorithm
+to check for connected vertices
+"""
+function quick_find(nodes, p, q)
+    pid = nodes[p]
+    qid = nodes[q]
+    for i in 1:length(nodes)
+        if nodes[i] == pid
+            nodes[i] = qid
+        end
+    end
+end
+
+function check_connected(nodes, p, q)
+
+    return nodes[p] == nodes[q]
+end
+
+"""Kruskal's algorithm
+to find minimum spanning tree for undirected graphs
+"""
+function kruskal_mst{T}(
+    g::SimpleGraph,
+    distmx::AbstractArray{T, 2}
+)
+
+    edge_list = []
+    mst = []
```

Comment:
What type is this? Right now you're creating a vector of `Any` which is not optimal. From below, it looks like this is also `Vector{Edge}()`.